README.md

```diff
@@ -57,6 +57,9 @@ See instructions on the [GitHub release page](https://github.com/catppuccin/user
    - Replacing `<name>` with the name of the userstyle.
 3. Enjoy!
 
+> [!TIP]
+> For decentralized or self-hosted apps applications, you can save your instance long term by going to the userstyle's **Settings > Custom included sites** and adding your instance to the list, for example, `*://example.org/*`.
```

Comment:
```suggestion
> For decentralized or self-hosted applications, you can apply a theme your instance across updates by going to the userstyle's **Settings** > **Custom included sites** and adding your instance to the list.
```
I don't think we need an example since the placeholder text of the input field gives the same example.